jobs/integr8ly/enmasse-address-creation-test.yaml

```diff
@@ -0,0 +1,73 @@
+---
+
+- job:
+    name: enmasse-address-creation-test
```

Comment:
Maybe better to use AMQ Online? WDYT? I don't have a strong opinion here but this might be confusing for newcomers

Reply:
done. It can be verified here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Integr8ly/job/amq-online-address-creation-test/2/console ( I will probably need to change the test to wait 90 instead of 60 seconds for the address to be ready, because every now and then the test itself fails (not related to the pipeline))

Reply:
ok. I was thinking about some global TIMEOUT variable in our testsuite (so that it can be increased globally) and we can e.g multiply this value in each test accordingly. I think I will sum it up in some JIRA

Reply:
Thanks anyway

Reply:
so are there any other changes required for this PR?

Reply:
One more update - add the new pipeline to the script for pipelines recreation please.

Reply:
Good catch! Done